Tests/MountebankSwiftTests/Api/MountebankTests+ErrorHandling.swift

```diff
@@ -120,7 +120,7 @@ extension MountebankTests {
                 completionExpectation.fulfill()
             }
 
-            wait(for: [completionExpectation], timeout: 1)
+            await fulfillment(of: [completionExpectation], timeout: 1)
         }
```

Comment:
I think this can become 

```
            do {
                let allImposters = try await sut.getAllImposters()
                XCTFail("Did expect failure but recieved success \(allImposters)")
            } catch {
                assertResult(error)
            }